examples/test-programs/job-queue/step1/index.ts

```diff
@@ -0,0 +1,87 @@
+import * as aws from "@pulumi/aws";
```

Comment:
I was working on creating an upgrade test for the job-queue upgrade, but I couldn't get it to work. Postponing to a follow up task.